k-distribution/include/builtin/kast.k

```diff
@@ -196,6 +196,12 @@ module RULE-LISTS
 
 endmodule
 
+module DEFAULT-LAYOUT
+    syntax Layout ::= r"(\\/\\*([^\\*]|(\\*+([^\\*\\/])))*\\*+\\/)"
```

Comment:
This should be #Layout.

Reply:
Looking at `kore/.../Sorts.scala`, it uses `K_` prefix instead of `#`. Should we pick `KLayout` instead for consistency?